### DISCUSSION TITLE
SyncWord behavior in Lora

### DISCUSSION THREAD

**Discussion by IbrahimMuala** [jgromes/RadioLib#634] [Q&A] [answered]
Hello,
Microcontroller: ESP32
Module: RFM95 Lora
Frequency: 915 MHz

I have no idea if someone asked this question before. I have used some other libraries for sending data using Lora from some time ago.

I was having 24 ESP32s, and they are moving (like someone hold them and running), I used another ESP32 as a controller for these 24 ESPs, every ESP of these is configured as unique SyncWord, and I was changing the SyncWord in the controller ESP32 in order to achieve direct communication between the controller and a certain ESP32. But, most of the time some of another ESPs were taking this information.

My question is, if I use this library, is this weird behavior will occurred? if yes is there any way to achieve direct and broadcast communication between the controller and the 24 ESP32s?

Thanks.

**Comment by jgromes** [comment, marked as answer]
What you are referring to as "weird behavior" is actually a result of the way sync word in LoRa is implemented - it's never going to be a 100% filter. It can also change receiver sensitivity, so it's not advised to use different values than the ones recommended in datasheets (Semtech's words, not mine). So you will have to find some other way ti implement node address in your system (since that's what it seems like you are trying to ultimately achieve).
